app/src/main/res/values-ja/strings.xml

```diff
@@ -574,6 +574,8 @@
     <string name="url_copied">URL をコピーしました</string>
 
 
+    <!-- Sample text for accessiblity font size -->
+    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.</string>
```

Comment:
Should the Japanese accessibility text only be in English?

Reply:
TBH, I don't know why it's labelled a11y. Is https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/layout_percentage_seek_bar.xml#L75 a11y-only?

Reply:
(also, typo in that comment…)

Reply:
As always, that's an upstream bug, we copy comments from English.